modules/extensions-spring-support/src/test/java/org/mule/runtime/module/extension/internal/resources/OCSExtensionModelJsonGeneratorTestCase.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.module.extension.internal.resources;
+
+import static org.mule.runtime.extension.internal.ocs.OCSConstants.OCS_ENABLED;
+import static org.mule.runtime.module.extension.internal.resources.ExtensionModelJsonGeneratorTestCase.ExtensionJsonGeneratorTestUnit.newTestUnit;
+
+import org.mule.runtime.extension.api.loader.ExtensionModelLoader;
+import org.mule.runtime.module.extension.api.loader.java.DefaultJavaExtensionModelLoader;
+import org.mule.tck.junit4.rule.SystemProperty;
+import org.mule.test.oauth.TestOAuthExtension;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.runners.Parameterized;
+
+public class OCSExtensionModelJsonGeneratorTestCase extends ExtensionModelJsonGeneratorTestCase {
+
```

Comment:
The property needs to also be present for the deserialization test.

Reply:
What do you mean?

Reply:
Ignore this comment. I was wrong, I meant the load test, but it is not necessary.